docs/src/concepts/fine-tuning.rst

```diff
@@ -0,0 +1,217 @@
+
+.. code-block:: yaml
+
+  training_set:
+      systems:
+        read_from: path/to/dataset.xyz
+        length_unit: angstrom
+      targets:
+        energy/<variantname>:
+          quantity: energy
+          key: <energy-key>
+          unit: <energy-unit>
+          description: "description of your variant"
+
+
+The new energy variant can be selected for evaluation either with ``mtt eval`` by specifying
+it in the options.yaml for evaluation:
+
+.. code-block:: yaml
+
+   systems: path/to/dataset.xyz
+   targets:
+     energy/<variantname>:
+       key: <energy-key>
+       unit: <energy-unit>
+       forces:
+         key: forces
+
+
+When using the finetuned model in simulation engines the default target name expected
+by the ``metatomic`` package in order to use the model in ASE and LAMMPS calculations is
+``energy``. When loading the model in ``metatomic`` you have to specify which variant
+should be used for energy and force prediction. You can find an example for how to do
+this in the tutorial :ref:`Fine-tuning <fine-tuning-example>` and more in the
+`metatomic documentation`_.
+
+.. _metatomic documentation: https://docs.metatensor.org/metatomic/latest/engines/index.html
+
+
+Until here, our model would train on all weights of the model, create a new energy head
+and a new composition model.
+
+The basic fine-tuning strategy is a good choice for most use cases. Below, we present
+a few more advanced topics.
+
+Inheriting weights from existing heads
+--------------------------------------
+
+In some cases, the new targets might be similar to the existing targets
+in the pre-trained model. For example, if the pre-trained model is trained
+on energies and forces computed with the PBE functional, and the new targets
+are energies and forces coming from the PBE0 calculations, it might be beneficial
+to initialize the new PBE0 heads and last layers with the weights of the PBE
+heads and last layers. This can be done by specifying the ``inherit_heads``
+parameter in the ``options.yaml`` file:
+
+.. code-block:: yaml
+
+  architecture:
+    training:
+      finetune:
+        method: full
+        read_from: path/to/checkpoint.ckpt
+        inherit_heads:
+          energy/<variantname>: energy # inherit weights from the "energy" head
+
+The ``inherit_heads`` parameter is a dictionary mapping the new trainable
+targets specified in the ``training_set/targets`` section to the existing
+targets in the pre-trained model. The weights of the corresponding heads and
+last layers will be copied from the source heads to the destination heads
+instead of random initialization. These weights are still trainable and
+will be adapted to the new dataset during the training process.
+
+
+Multi-fidelity training
+-----------------------
+Even though the old head is left untouched, it is rendered useless, due to changing
+deeper weights of the model. If you want to fine-tune and retain multiple functional
```

Comment:
This sentences comes a little out of nowhere. Maybe something connecting the last sections.


```suggestion
So far the old head is left untouched, but it is rendered useless, due to changing
deeper weights of the model. If you want to fine-tune and retain multiple functional
```